src/example.wordswipe-test.config.js

```diff
@@ -51,71 +51,8 @@ export default {
     name: "openmoji",
     url: "http://openmoji.org",
   },
-
-  levels: {
-    0: {
-      level: 0,
-      min: 0,
-      max: 100,
-      character: null,
-      img: null,
-      img_grey: null,
-    },
-    1: {
-      level: 1,
-      min: 20,
-      max: 50,
-      character: "fish",
-      img: "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/color/svg/1F41F.svg?sanitize=true",
-      img_grey:
-        "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/black/svg/1F41F.svg?sanitize=true",
-    },
-    2: {
-      level: 2,
-      min: 51,
-      max: 100,
-      character: "frog",
-      img: "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/color/svg/1F438.svg?sanitize=true",
-      img_grey:
-        "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/black/svg/1F438.svg?sanitize=true",
-    },
-    3: {
-      level: 3,
-      min: 101,
-      max: 150,
-      character: "duck",
-      img: "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/color/svg/1F986.svg?sanitize=true",
-      img_grey:
-        "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/black/svg/1F986.svg?sanitize=true",
-    },
-    4: {
-      level: 4,
-      min: 151,
-      max: 200,
-      character: "koala",
-      img: "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/color/svg/1F428.svg?sanitize=true",
-      img_grey:
-        "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/black/svg/1F428.svg?sanitize=true",
-    },
-    5: {
-      level: 5,
-      min: 201,
-      max: 250,
-      character: "cat",
-      img: "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/color/svg/1F408.svg?sanitize=true",
-      img_grey:
-        "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/black/svg/1F408.svg?sanitize=true",
-    },
-    6: {
-      level: 6,
-      min: 2001,
-      max: 2500,
-      character: "lion",
-      img: "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/color/svg/1F981.svg?sanitize=true",
-      img_grey:
-        "https://raw.githubusercontent.com/hfg-gmuend/openmoji/master/black/svg/1F981.svg?sanitize=true",
-    },
-  },
+  // Each value denotes the minimum amount of cumulative points needed to attain that level
+  levels: [20, 50, 100, 150, 200, 300, 500, 800, 1200],
```

Comment:
Might be good to use the key `badges` or `achievements` for this so to not conflict with the idea of a level within a game? 
Maybe something like
```js
badgeUnlockScores: [20, 50, ...]
```

Reply:
Hey thanks, i think badgeUnlockScores is a good name for it. I used levels at first because of the previous naming in the config file, but it's more like an "achievement"